meta/saimetadatatypes.h

```diff
@@ -578,6 +652,13 @@ typedef enum _sai_attr_flags_t
      */
     SAI_ATTR_FLAGS_SPECIAL             = (1 << 6),
 
+    /**
+     * @brief Set only flag.
+     *
+     * Attribute with this flag can only be wrote using SET API. Read is not
+     * possible.
+     */
+    SAI_ATTR_FLAGS_SET_ONLY           = (1 << 7),
```

Comment:
SAI_ATTR_FLAGS_SET_ONLY flag seems to be ood, in this case we dont need special flag, just return not supported status when doint READ on attribute, since api will be there anyway and user can still do GET